snmp/tests/test_e2e_core_profiles/test_profile_riverbed_steelhead.py

```diff
@@ -0,0 +1,96 @@
+    get_device_ip_from_config,
+)
+
+pytestmark = [pytest.mark.e2e, common.py3_plus_only, common.snmp_integration_only]
+
+
+def test_e2e_profile_riverbed_steelhead(dd_agent_check):
+    config = create_e2e_core_test_config('riverbed-steelhead')
+    aggregator = common.dd_agent_check_wrapper(dd_agent_check, config, rate=True)
+
+    ip_address = get_device_ip_from_config(config)
+    common_tags = [
+        'snmp_profile:riverbed-steelhead',
+        'snmp_host:riverbed-steelhead.device.name',
+        'device_namespace:default',
+        'snmp_device:' + ip_address,
+    ] + []
+
+    # --- TEST EXTENDED METRICS ---
+    assert_extend_generic_if(aggregator, common_tags)
+
+    # --- TEST METRICS ---
+    assert_common_metrics(aggregator, common_tags)
+
+    aggregator.assert_metric('snmp.cpu.usage', metric_type=aggregator.GAUGE, tags=common_tags)
+    aggregator.assert_metric('snmp.memory.total', metric_type=aggregator.GAUGE, tags=common_tags)
+    aggregator.assert_metric('snmp.memory.usage', metric_type=aggregator.GAUGE, tags=common_tags)
+    aggregator.assert_metric('snmp.memory.used', metric_type=aggregator.GAUGE, tags=common_tags)
+    aggregator.assert_metric(
+        'snmp.riverbed.steelhead.activeConnections', metric_type=aggregator.GAUGE, tags=common_tags
+    )
+    aggregator.assert_metric(
+        'snmp.riverbed.steelhead.establishedConnections', metric_type=aggregator.GAUGE, tags=common_tags
+    )
+    aggregator.assert_metric(
+        'snmp.riverbed.steelhead.halfClosedConnections', metric_type=aggregator.GAUGE, tags=common_tags
+    )
+    aggregator.assert_metric(
+        'snmp.riverbed.steelhead.halfOpenedConnections', metric_type=aggregator.GAUGE, tags=common_tags
+    )
+    aggregator.assert_metric(
+        'snmp.riverbed.steelhead.optimizedConnections', metric_type=aggregator.GAUGE, tags=common_tags
+    )
+    aggregator.assert_metric(
+        'snmp.riverbed.steelhead.passthroughConnections', metric_type=aggregator.GAUGE, tags=common_tags
+    )
+    aggregator.assert_metric('snmp.riverbed.steelhead.totalConnections', metric_type=aggregator.GAUGE, tags=common_tags)
+
+    aggregator.assert_metric('snmp.ifBandwidthInUsage.rate')
+    aggregator.assert_metric('snmp.ifBandwidthOutUsage.rate')
+    aggregator.assert_metric('snmp.ifHCInBroadcastPkts')
+    aggregator.assert_metric('snmp.ifHCInMulticastPkts')
+    aggregator.assert_metric('snmp.ifHCInOctets')
+    aggregator.assert_metric('snmp.ifHCInOctets.rate')
+    aggregator.assert_metric('snmp.ifHCInUcastPkts')
+    aggregator.assert_metric('snmp.ifHCOutBroadcastPkts')
+    aggregator.assert_metric('snmp.ifHCOutMulticastPkts')
+    aggregator.assert_metric('snmp.ifHCOutOctets')
+    aggregator.assert_metric('snmp.ifHCOutOctets.rate')
+    aggregator.assert_metric('snmp.ifHCOutUcastPkts')
+    aggregator.assert_metric('snmp.ifHighSpeed')
+    aggregator.assert_metric('snmp.ifInSpeed')
+    aggregator.assert_metric('snmp.ifOutSpeed')
```

Comment:
same as for the [other PR](https://github.com/DataDog/integrations-core/pull/15294#),  you need to remove all entries starting with `1.3.6.1.2.1.31.1.*` from your snmprec to be able to remove these tests